scripts/plugins/utilities.py

```diff
@@ -369,9 +372,8 @@ def run_cmd(command: Union[list, str]) -> subprocess.CompletedProcess:
             # about the status of the process execution.
             # Namely the return code which is used elsewhere.
             process.communicate()
+
     except Exception as exc:
         logger.error(exc)
-        if BITOPS_fast_fail_mode:
-            sys.exit(101)
-
+        sys.exit(101)
     return process
```

Comment:
The only thing we need to ensure the correct data type is specified here: 

https://github.com/bitovi/bitops/blob/dab9410f793a56a37105f6232e11c6616432a3a8/scripts/plugins/utilities.py#L352-L353

I think it's `subprocess.Popen` now.

Reply:
**Log**
`2022-12-09 14:13:05,519 root         CRITICAL <class 'subprocess.Popen'>`

**Tested with**
```
process.communicate()
logger.critical(type(process))
```

I agree with the change 👍